### ISSUE TITLE
Unescaped quotes of dotenv files by Config class

### ISSUE THREAD

**Issue by yoagauthier** [FoalTS/foal#788]
Hello !

## TLDR
Foal `Config` class reads `.env` file to access some config variables.
However, it seems to not fully comply with [others](https://github.com/motdotla/dotenv) (official ?) `.env` implementations when parsing the content.

## Reproducing
For instance, when my `.env` file is :
```env
MY_VAR="test"
```
when performing
```
const myVar = Config.get2('myVar');
```
the result is `"test"` instead of `test`

The [dotenv rules](https://github.com/motdotla/dotenv#rules) specifies that _"single and double quoted values are escaped"_

This [function](https://github.com/FoalTS/foal/blob/master/packages/core/src/core/config/config.ts#L210) indeed does not seem to escapce those beginning / ending quotes.

Is this an issue or a design choice ?

If that's an issue, it could be a breaking change for v2 (I could do the PR to correct this, but only in September, and that might be late for v2 ...)

**Comment by LoicPoullain** [comment]
Hi @yoagauthier 

Thank you for your suggestion. Single and double quotes will be supported in v2 (PR merged).
